### DISCUSSION TITLE
How can I get the Arctic Code Vault Contributor and Mars Helicopter badge on my profile?

### DISCUSSION THREAD

**Discussion by odin0809** [community/community#20919] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello All, I am new to github. I want to know how to get the Arctic Code Vault Contributor and Mars Helicopter Contributor badge on my profile? Someone told me that this badge I can't get now. Is it true?

**Comment by radar-blip** [comment]
True. You cant get those anymore.
Here is more info on achievement badges:
https://github.com/Schweinepriester/github-profile-achievements#achievements

They were only available during certain months in 2020

**Comment by Chandan-Choudhury** [comment, marked as answer]
Hello @odin0809 , unfortunately you can't earn these badges now. But you can now earn other badges like **Starstruck**, **Pair Extraordinaire**, **Quickdraw**, **YOLO**, **Galaxy Brain** & **Public Sponsor**  to your profile. You have to complete some tasks/related work to earn these achievements badges to your profile.

Checkout this link for more information : https://github.com/drknzz/GitHub-Achievements
